coalib/output/ConsoleInteraction.py

```diff
@@ -547,6 +547,7 @@ def require_setting(setting_name, arr, section):
                          of the bears who need this setting in [1] and
                          following.
     :param section:      The section the action corresponds to.
+    :param return:       Takes input of the missing setting from the user.
```

Comment:
The `return` takes input? That's kind of weird :D
Probably you want to say it returns the setting value that was requested from the user ;)

Reply:
Sorry, I didn't frame it properly 🤣. Gonna fix it asap.